housekeeper/cli/add.py

```diff
@@ -70,6 +72,13 @@ def bundle_cmd(context: click.Context, bundle_name: str, json: str):
     store.session.add(new_bundle)
     new_version.bundle: Bundle = new_bundle
     store.session.add(new_version)
+    if not exclude:
+        try:
+            include_version(context.obj[ROOT], new_version)
```

Comment:
What is `context.obj[ROOT]`? I found this confusing, maybe it can be made clearer somehow

Reply:
Clarified with variable instantiation and named arguments.